webapp/src/main/java/com/box/l10n/mojito/entity/ScheduledJobTypeEntity.java

```diff
@@ -0,0 +1,46 @@
+package com.box.l10n.mojito.entity;
+
+import static org.hibernate.envers.RelationTargetAuditMode.NOT_AUDITED;
+
+import com.box.l10n.mojito.rest.View;
+import com.box.l10n.mojito.service.scheduledjob.ScheduledJobType;
+import com.fasterxml.jackson.annotation.JsonView;
+import jakarta.persistence.Basic;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import org.hibernate.envers.Audited;
+
+@Entity
+@Table(name = "scheduled_job_type")
+@Audited(targetAuditMode = NOT_AUDITED)
+public class ScheduledJobTypeEntity extends BaseEntity {
```

Comment:
For the actual entity class names - we don't tend to suffix them with the word "Entity", e.g.:
`public class Locale extends BaseEntity` (vs. `LocaleEntity`)

Reply:
I ended up using Entity for the suffix here as I have another class labelled ScheduledJobType in another package that I created (You will see it further on in the PR). Would the best approach be to rename that ScheduledJobType to something else so that the entity version can use it ? I'm thinking of calling the current one : ScheduledJobTypeEnum as the enum currently uses that name.

Reply:
This is done - @garionpin